tfdeterminism/patch.py

```diff
@@ -70,13 +70,20 @@ def _patch():
   if re.match("(1\.(14|15)|2\.0)", tf_version):
     os.environ['TF_CUDNN_DETERMINISTIC'] = '1'
     _patch_bias_add()
+    _patch_fused_softmax_cross_entropy()
```

Comment:
I'll push changes that rough-out `enable_determinism` and then you can call this from the appropriate part of that.